### DISCUSSION TITLE
Vercel is creating a new project each time I push my repo

### DISCUSSION THREAD

**Discussion by BuchananDevOps** [vercel/community#1409] [Help]
Dear Vercel DevRel's, 

This morning I noticed multiple new projects in my account but they were all linked to the same repo. It seems that for each git push yesterday Vercel went about automatically creating a new project for each. This morning I had 5 projects all linked to the same repo. While it isn't cause for concern I was able to simply delete the additional repo's, I just wanted to give you the heads up that this is happening.

Could it be a setting on my end causing this?

**Comment by amyegan** [comment]
Thanks for reaching out @BuchananDevOps. Is your repo public? Or can you share a deployment link or [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) so we can get a better look at what is causing the problem?
